### ISSUE TITLE
Consider adding language specific resources for implementing these principles

### ISSUE THREAD

**Issue by michaelsearcy** [interagent/http-api-design#68]
I tried to implement some of these principles and found that it involved certain other resources. For ASP.net for example it seems like [CacheCow](https://github.com/aliostad/CacheCow/wiki) is the best available way to implement ETags for Caching. Maybe adding some links to similar resources for specific server/client languages would serve as a good jumping off point for people who find this useful.

**Comment by geemus** [comment]
Good idea! I'm hardly an expert outside of my pretty narrow (Ruby mostly) realm, but I can certainly see how some pointers would be helpful. I'll ponder how best to fit this in (I don't want it to be too overwhelming or distracting from the core content). Thanks for the suggestion!
